system-test/bigtable.js

```diff
@@ -57,7 +60,20 @@ describe('Bigtable', function() {
             {
               families: ['follows', 'traits'],
             },
-            done
+            function(err) {
```

Comment:
Can you please test the `routing: cluster` path?

Reply:
We test that in setMetadata https://github.com/googleapis/nodejs-bigtable/pull/124/files#diff-107134a4c52159839ec2ea9b9fb78460R211

Reply:
ack